packages/slice-machine/components/ScreenshotPreview/index.tsx

```diff
@@ -8,45 +8,42 @@ const MemoedImage = memo<{ src: string | undefined }>(({ src }) => (
 
 interface ScreenshotPreviewProps {
   src?: string;
-  sx?: ThemeUIStyleObject;
+  sx: { height: string } & ThemeUIStyleObject;
```

Comment:
Typescript question,

If you do `sx: { height: string } & ThemeUIStyleObject;` does this mean the height property in the final object is the one coming from ThemeUISTyleObject ?

In comparison with `sx: ThemeUIStyleObject & { height: string }` where you would extend the ThemeUIStyleObject.

It's a real question, i'm not sure of the answer 😄

Reply:
I don't think the order matters in type unions (A & B === B & A). In this case all it's doing is making height required since it has the same type in both objects.

The union just forces the type to match both types.

Reply:
Actually just realised the ThemeUI type is `string | number`, will update it now so the number type can also be used

Reply:
Ok so it match both types, always thought it was to change properties of one given type. 👍

Reply:
thanks :)